### ISSUE TITLE
Issue after #1784 commit $port_min, $port_max

### ISSUE THREAD

**Issue by Psychoses** [AzuraCast/AzuraCast#1799]
Hi,

i have a traditional/ansible install.

i have no problem with 0.9.5.1
0.9.6 give me "no sound on public page" who seems fixed with #1790 
i can get this work until here.

after #1784 a lot of problem appear.
the problem seems come from src\Radio\Configuration.php
$port_min, $port_max

```
        $port_min = (int)($_ENV['AUTO_ASSIGN_PORT_MIN'] ?? 8000);
        $port_max = (int)($_ENV['AUTO_ASSIGN_PORT_MAX'] ?? 8499);

		echo $port_min;
		// 8000
		echo $port_max;
		// 8499

        $port_min = (int)(getenv('AUTO_ASSIGN_PORT_MIN') ?? 8000);
        $port_max = (int)(getenv('AUTO_ASSIGN_PORT_MAX') ?? 8499);

		echo $port_min;
		// 0
		echo $port_max;
		// 0
```

**Comment by SlvrEagle23** [comment]
@Psychoses Should be fixed as of the latest commit. Thanks for letting us know.

**Comment by Psychoses** [comment]
should be ok. thx u !

**Comment by Psychoses** [comment]
@SlvrEagle23 tested ! all works like expected from my point of view. my station is on air ^^
v0.9.6, #f06a6e2 (2019-07-30 14:19) • Ansible • PHP 7.2

**Comment by hillbilly-mark** [comment]
I jave the same issue as #1784 which is now closed.  The solution made zero sense.  Where do you find this white space or other settings mentioned?

My docker run on ubuntu 18.04.

**Comment by SlvrEagle23** [comment]
@hillbilly-mark Are you in the process of trying to change which port number your station is streaming on? That would be important for you to note in your other issue. In any case, you should make a new issue if you're experiencing problems with custom port mappings.
